run-integration-tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 function usage {
-    echo "Usage: source ./$( basename "$( readlink -f "${BASH_SOURCE[0]}" )" ) [-s <maven_settings_file>]"
+    echo "Usage: source ./$( basename "$( python -c "import os; print os.path.abspath('${BASH_SOURCE[0]}')" )" ) [-s <maven_settings_file>]"
```

Comment:
These changes should not be here since a rebase doesn't introduce a merge commit. Should be correctable via a `git reset --hard` back to the commit one before the erroneous merge commit and force-pushing?